backend/Data/CertificateManagerDbContext.cs

```diff
@@ -30,7 +30,8 @@ public CertificateManagerDbContext(DbContextOptions<CertificateManagerDbContext>
 
     public virtual DbSet<User> Users { get; set; }
 
-
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
```

Comment:
This method should not use here, all connection string configuration should be in Program.cs